src/main/java/spoon/support/reflect/code/CtStatementImpl.java

```diff
@@ -183,7 +183,13 @@ void insert(CtBlock<?> block, CtStatementList statementsToBeInserted) {
 
 			@Override
 			void insertFromFirstStatement(CtBlock<?> block, CtStatement target, CtStatementList statementsToBeInserted) {
-				int indexOfTargetElement = block.getStatements().indexOf(target);
+			  int indexOfTargetElement = -1;
```

Comment:
Hi,

I just tried to use the formatter included with the spoon' repository ( SpoonFormatterPrefs.xml ), however when I apply it to the project, 458 files are modified. Do you really use this formatter ?